src/renderers/WebGLRenderer.js

```diff
@@ -2779,18 +2779,14 @@ class WebGLRenderer {
 				const dstRenderTargetProperties = properties.get( dstTextureProperties.__renderTarget );
 
 				state.bindFramebuffer( _gl.READ_FRAMEBUFFER, srcRenderTargetProperties.__webglFramebuffer );
-
-				_gl.framebufferTextureLayer( _gl.READ_FRAMEBUFFER, _gl.COLOR_ATTACHMENT0, properties.get( srcTexture ).__webglTexture, level, 0 );
-
 				state.bindFramebuffer( _gl.DRAW_FRAMEBUFFER, dstRenderTargetProperties.__webglFramebuffer );
 
-				_gl.framebufferTextureLayer( _gl.DRAW_FRAMEBUFFER, _gl.COLOR_ATTACHMENT0, properties.get( dstTexture ).__webglTexture, level, 0 );
+				for ( let i = 0; i < depth; i ++ ) {
 
-				let mask = _gl.COLOR_BUFFER_BIT;
+					_gl.framebufferTextureLayer( _gl.READ_FRAMEBUFFER, _gl.COLOR_ATTACHMENT0, properties.get( srcTexture ).__webglTexture, level, minZ + i );
+					_gl.copyTexSubImage3D( glTarget, level, dstX, dstY, dstZ + i, minX, minY, width, height );
 
-				if ( srcTexture.isDepthTexture ) mask = _gl.DEPTH_BUFFER_BIT;
-
-				_gl.blitFramebuffer( minX, minY, width, height, dstX, dstY, width, height, mask, _gl.NEAREST );
```

Comment:
If you completely remove the usage of `blitFramebuffer()`, the method does not support depth attachments anymore. That means the `srcTexture.isDepthTexture` should be removed and the documentation should be updated.

Reply:
Is it possible to have a depth buffer attachment for a 3d texture?

Reply:
I believe so. How would depth testing work otherwise?

Reply:
I guess I hadn't really considered the case where you'd be rendering to volume textures that way. I can add it back as an option.

To be clear it doesn't look like the previous code path can ever copy the depth buffer for a color render target in the 2d or 3d case:

```js
let mask = _gl.COLOR_BUFFER_BIT;

if ( srcTexture.isDepthTexture ) mask = _gl.DEPTH_BUFFER_BIT;
```

The depth component is only copied when `isDepthTexture` is on the src texture which will never be the case with normal render render targets. We also have no method for creating 3d depth textures. Is this the kind of logic we want instead?

```js
let mask = _gl.COLOR_BUFFER_BIT | _gl.DEPTH_BUFFER_BIT | _gl.STENCIL_BUFFER_BIT;

if ( srcTexture.isDepthTexture ) mask = _gl.DEPTH_BUFFER_BIT;
```

ie just copy all the attached buffers unless it's a depth texture? I feel we might as well just set all the bits anyway regardless of target type.

Reply:
> The depth component is only copied when isDepthTexture is on the src texture which will never be the case with normal render render targets.

We already use the following pattern and it works as expected:
```js
renderer.copyTextureToTexture( this.sampleRenderTarget.depthTexture, this.renderTarget.depthTexture );
```

I'm okay with keeping the PR as it is and revisit the issue when a developer requests it. For the normal `copyTextureToTexture()`, the support is critical though.

Reply:
As long as the original blit is retained for the depth texture, I don't think something should break.

Reply:
Okay! In that case this should be ready if it looks good to you.